sdk/azidentity/confidential_client.go

```diff
@@ -118,7 +118,7 @@ func (c *confidentialClient) GetToken(ctx context.Context, tro policy.TokenReque
 		msg := fmt.Sprintf(scopeLogFmt, c.name, strings.Join(ar.GrantedScopes, ", "))
 		log.Write(EventAuthentication, msg)
 	}
-	return azcore.AccessToken{Token: ar.AccessToken, ExpiresOn: ar.ExpiresOn.UTC()}, err
+	return azcore.AccessToken{Token: ar.AccessToken, ExpiresOn: ar.ExpiresOn.UTC(), RefreshOn: ar.Metadata.RefreshOn.UTC()}, err
```

Comment:
I assume `RefreshOn.UTC()` returns the zero-value if there's no `RefreshOn` yes?

Reply:
yes, `UTC()` for zero is still zero, so the call has no observable effect when MSAL didn't set a value for one of these Times